source/docs/advanced/theme/analytics.md

```diff
@@ -295,6 +295,26 @@ export default {
 };
 ```
 
+The consent for the cookies is stored in 2 cookies:
+
+1. `hasConsent` cookie which stores if the user provided consent answer (authorized or denied) for all services.
+2. `authorizations` cookie which stores a JSON string of all consents given in the following format `{ [service1_name]: true | false, ... }`
+
+These two cookies' expiration SHOULD be configured in `src/config/website.js` like:
+
+```diff
+module.exports = {
+  default_image_url: "https://placehold.it/150x220",
+  available_page_sizes: [18, 36],
+  ....
+  rewrittenToRoot: ["/home"],
+  useUserTitles: false,
++  cookieMaxAgeInMonths: 13,
+};
+```
+
+As of `2.12.0` and patch releases `2.11.1`, `2.10.3`, `2.9.5`, `2.8.6`, `2.7.4`, `2.6.3`, `2.5.5`, `2.5.5`, `2.4.9` the default will be 13 month but if you are on Front-Commerce version less than the ones above setting up `cookieMaxAgeInMonths` in your `src/config/website.js` IS A MUST. Otherwise you run the risk of the cookie banner showing to your user everytime he closes and opens his browser. Which is very annoying to the user and may lead to the user denying consent and thus the loss valuable analitics insights.
```

Comment:
I agree. It should be in migration guide.
Patch releases will contain it in their changelog and we may do a specific section in the release announcement about it.

But I don't think the documentation should go into details here.

Reply:
done in [pr270](https://github.com/front-commerce/developers.front-commerce.com/pull/270)

Reply:
i still think this should be kept in the `configuration` file otherwise the migration guide ppl only read if they want to migrate, and we'd have to wait to 2.12.0 release in 6 weeks to have next merged making any reference here to the migration guide broken in the meantime :S

Reply:
No strong opinion on my side. The important is that it is part of the migration guide.